src/edit/items/renderPieceIdentifiers.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'redux-form';
+import { FormattedMessage } from 'react-intl';
+import { Row, Col, Button, TextField } from '@folio/stripes/components';
+
+
+const renderPieceIdentifiers = ({ fields, meta: { touched, error, submitFailed } }) => {
+  const handleClickAddIdentifier = () => fields.push();
+
+  return (
+    <div>
+      <Row>
+        <Col sm={2} smOffset={5}>
+          <Button
+            type="button"
+            buttonStyle="fullWidth secondary"
+            id="clickable-add-notes"
+            onClick={handleClickAddIdentifier}
+          >
+            <FormattedMessage id="ui-inventory.addPieceIdentifier" />
+          </Button>
+          {(touched || submitFailed) && error && <span>{error}</span>}
+        </Col>
+      </Row>
+      {fields.map((pieceIdentifier, index) => (
+        <Row key={index}>
```

Comment:
`index` is a bad value for `key`. Can anything more unique. Can be used instead?